backend/src/main/java/com/redhat/cloud/notifications/processors/email/aggregators/DriftEmailPayloadAggregator.java

```diff
@@ -0,0 +1,88 @@
+package com.redhat.cloud.notifications.processors.email.aggregators;
+
+import com.redhat.cloud.notifications.models.EmailAggregation;
+import com.redhat.cloud.notifications.models.EmailAggregationKey;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+public class DriftEmailPayloadAggregator extends AbstractEmailPayloadAggregator {
+
+    private static final String DRIFT_KEY = "drift";
+    private static final String HOST_KEY = "hosts";
+    private static final String UNIQUE_SYSTEM_COUNT = "unique_system_count";
+    private static final String CONTEXT_KEY = "context";
+    private static final String EVENTS_KEY = "events";
+    private static final String PAYLOAD_KEY = "payload";
+
+    // Drift Payload
+    private static final String BASELINE_ID = "baseline_id";
+    private static final String BASELINE_NAME = "baseline_name";
+
+    private static final String DISPLAY_NAME = "display_name";
+    private static final String INVENTORY_ID = "inventory_id";
+    private static final String TAGS = "tags";
+
+    private final Set<String> uniqueHosts = new HashSet<>();
+    private final Map<String, HashSet<String>> uniqueHostPerBaseline = new HashMap<>();
+
+    static AbstractEmailPayloadAggregator by(EmailAggregationKey aggregationKey) {
+        if (aggregationKey.getBundle().equals("rhel") && aggregationKey.getApplication().equals("drift")) {
+            return new DriftEmailPayloadAggregator();
+        }
+
+        return null;
```

Comment:
We don't need this function here, and the content has to be in the [EmailPayloadAggregatorFactory.java](https://github.com/RedHatInsights/notifications-backend/blob/master/backend/src/main/java/com/redhat/cloud/notifications/processors/email/aggregators/EmailPayloadAggregatorFactory.java#L15) file